recipes/libflint/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.5.3.dev0" %}
+{% set commit = "6e1c767c29b5d1c0e94d33512181ad63e73f5fc7" %}
```

Comment:
I have raised this issue ( https://github.com/wbhart/flint2/issues/275 ) to see if we can get a new release.